sormas-e2e-tests/src/test/resources/features/sanity/web/Event.feature

```diff
@@ -235,6 +235,16 @@ Feature: Create events
     And I select Dropped filter from quick filter
     And I click on the RESET FILTERS button
 
+  @issue=SORDEV-9426
+  Scenario: Filter for the report date of events
+    Given I log in with National User
+    And I click on the Events button from navbar
+    Then I click on Show more filters in Events
+    And I select Report Date among Event Reference Date options
+    And I fill in a date range
```

Comment:
how stable is this step? we have chances to get 0 results if the date is today and we don't have available data

Reply:
The range being set by this step starts a month ago and ends today. If there are 0 results for the given time range, then the step simply passes (as it should). This situation however is extremely unlikely, since other existing scenarios do create events with the present date of report, so there should always be data that yields some results. I suppose I could add additional steps that create one event that's specifically outside the time range and another that's within it to see if the first one doesn't show and the second one shows, but that would make this test a lot more complicated and a lot more prone to failure due to factors that are outside the scope of the original scenario. Please let me know what's your opinion on this.

Reply:
ok, clear now :D thank you